includes/utils.sh

```diff
@@ -256,15 +256,15 @@ then
 		if [ "${RET0}" != "" ]
 		then
 			RET2=`eval norminette $RET0 2>&1`
+			RET4=$?
 			echo "$RET2" > "$RETURNPATH"/.mynorminette
 			RET2=`cat .mynorminette | grep Error`
 			RET3=`cat .mynorminette | grep Warning`
-			RET4=`cat .mynorminette | grep "command not found"`
 			if [ "$RET2" == "" -a "$RET3" == "" -a "$RET4" == "" ]
```

Comment:
Yes, then look at the line I just commented (263): the test will never be truethy if you keep ´$RET4 == ""´

Yes, eval returns the exit status of the executed command :-) but I was asking if the assignation ´=´ has its own exit status. I will test it today